README.md

```diff
@@ -337,6 +338,36 @@ in [Command Line Arguments](#command-line-arguments).
 you to start your application with SSL. More details in 
 [SSL Configuration](#ssl-configuration).
 
+## Back interface
+
+[Back](src/main/java/org/takes/http/Back.java) interface is the back-end that
+is responsible for accepting the incoming requests through the `Socket`, 
+dispatching them to relevant `Takes` instance and printing the result of 
```

Comment:
@t-izbassar actually, only `BkBasic` can dispatch requests to `Take` instance, all other implementations knows nothing about Takes.